data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			choice
+				`	"That's okay. I didn't mind waiting."`
+					goto waiting
+				`	"I was abducted by the Vice Planetary-Governor."`
+			`	"That is no surprise." Hjlod says nonchalantly. "They alvays do thing like this. It matters not to us."`
+			`	You ask her if she's worried you've been compromised - that you might be working against the Skadenga's best interests. Hjlod shakes her head. "No. Future is set. Vhat vill, vill be. You are given task to tell us vhere to go, ve go vhere you say. The rest is just noise.`
+			label waiting
+			`	"Now", Hjlod says, "Tell me about Setstokkr. Did you find it?"`
+			choice
+				`	"I found nothing definitive."`
+					goto nothing
+				`	"I think I found traces of it on Windblain."`
+					goto nothing
+				`	"I found a strange box similar to the Setstokkr on Asgard."`
+			`	"Sounds like herbalist box. Ondurdis had one long before I vas born. Gave it to liar. It not important though, not Setstokkr."`
+			label nothing
+			`	"Sending big piece of vood through space probably not best idea Ondurdis had." Hjlod smirks. "Still, it does not matter so much what you find. Only how you felt. You saw the vorlds. Vhere should ve go?"`
+			choice
+				`	"Windblain."`
+				`	"Asgard."`
+					goto asgard
+			apply
+				set "Nifel"
+			`	You answer, "There is a nearly abandoned village called Hrithfjall next to a coastal God-Stone. Something about that place felt right. I believe that is where the Skadenga should migrate."`
+			`	Hjlod nods. "Hrithfjall sounds like good place. I vill tell Ondurdis.`
+			goto end
+			label asgard
+			apply
+				set "Asgard"
+			`	You answer, "On Asgard, there is a great market built around their God-Stone, and many new homes in need of people to fill them. The people there have lost their way and they need the Skadenga to remind them of who they were."`
+			`	Hjlod nods. "The Skadenga have alvays been guides, pathfinders. I vill tell Ondurdis.`
+			label end
+			`	"Vhen you were gone, thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
+			`	You frown and ask, "Did she provide money?"`
+			`	Hjlod shakes her head. "No. Ve could not afford it. You must go to Valhalla and convince head of Transport Union to support us." She squeezes your shoulder. "Thank you for help. For finding us new home. Ve need it. I must report now to Ondurdis. Skade be vith you." She turns and leaves.`
+				accept
+	on complete
+		conversation
+			`	Outside your ship, you are met by a representative of the Transport Union. "We have been expecting you, Captain <last>." They escort you to an elevator and up to the penthouse of the tallest building in Valhalla's capital of Himinbjorg."`
+			scene "scene/tower"
```

Comment:
This picture doesn't feel too tonally consistent with ES to me. I'd be curious what others besides myself think.